mockbee-test/ecom-store/src/pages/Products.js

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react";
+import { Navbar, ProductCard } from "../components";
+import axios from "axios"
+const Products = () => {
+    const [products, setProducts] = useState([]);
+    const [error, setError] = useState();
+
+    useEffect(() => {
+        (async () => {
+            try{
+                const {data : {products}} = await axios.get("/api/products")
+                setProducts(products);
+            }catch(error){
+                setError("No products to display");
```

Comment:
Where you are using this error state?

Reply:
I have not used it yet, but have kept it for further use.
Thank you for the review.